<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>leaderboard</class>
 <widget class="QDialog" name="leaderboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>leaderboard</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(222, 221, 218);
border: 3px solid rgb(0, 0, 0);
border-radius: 10px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="back_to_main_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 97, 81);</string>
    </property>
    <property name="text">
     <string>Back to the main window</string>
    </property>
   </widget>
   <widget class="QLabel" name="top_1_username_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(98, 160, 234);
border: 3px solid rgb(0, 0, 0);
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>1. username - points</string>
    </property>
   </widget>
   <widget class="QLabel" name="leaderboard_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(245, 194, 17);
border: 3px solid rgb(0, 0, 0);
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>                           Leaderboard</string>
    </property>
   </widget>
   <widget class="QLabel" name="top_2_username_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(98, 160, 234);
border: 3px solid rgb(0, 0, 0);
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>2. username - points</string>
    </property>
   </widget>
   <widget class="QLabel" name="top_3_username_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(98, 160, 234);
border: 3px solid rgb(0, 0, 0);
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>3. username - points</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
